portal-docs/observability.md

```diff
@@ -8,6 +8,8 @@
 * [Observe Application Metrics](#Observe-Application-Metrics)
 * [Tracing Requests](#Tracing-Requests)
```

Comment:
Shall we say "Trace Requests" so that we are following the present tense for all toc items?